Doc/library/pathlib.rst

```diff
@@ -7,7 +7,7 @@
 
 .. versionadded:: 3.4
 
-**Source code:** :source:`Lib/pathlib.py`
+**Source code:** :source:`Lib/pathlib/`
```

Comment:
This cannot be backported. I can move it to a separate commit or manually prepare PRs for backporting.

Reply:
It's ok to fix it in a manual backport.